src/sass/base/_flex-box.scss

```diff
@@ -20,4 +20,8 @@ flexBox column flexGrow_2
     justify-content: space-around;
 }
 
+.space-evenly {
```

Comment:
Another utility class... makes switching things around a snap, layout-wise. Nice and modular as well.